scripts/test-ci.sh

```diff
@@ -7,4 +7,5 @@ set -o pipefail
 
 # We have to use xvfb due to electron
 # Ref: https://github.com/facebook-atom/jest-electron-runner/issues/47#issuecomment-508556407
-xvfb-run -e /dev/stdout yarn workspace @mm-snap/controllers test
+xvfb-run -e /dev/stdout yarn workspace @mm-snap/controllers test &&
+yarn workspace @mm-snap/iframe-execution-environment-service test
```

Comment:
These are currently serial. Can they run in parallel?

```suggestion
xvfb-run -e /dev/stdout yarn workspace @mm-snap/controllers test &
yarn workspace @mm-snap/iframe-execution-environment-service test
```

Reply:
never mind